cmd/cmd-print.go

```
@@ -48,6 +48,8 @@ func print(cmd *cobra.Command, args []string) error {
 	strOutput, _ := flag.GetString("output")
 	strErrOutput, _ := flag.GetString("error-output")
 
+	_ = flag.Set("readOnly", "true")
```

ignoring the error here because we know `readOnly` flag is available.